src/views/Settings.vue

```diff
@@ -8,68 +8,168 @@
+          <ion-card-content>
+            {{ $t('Enter a custom date time format that you want to use when uploading documents to HotWax Commerce.') }}
+            <p>{{ $t('Luxon date time formats can be found') }} <a href="https://github.com/moment/luxon/blob/master/docs/formatting.md">{{ $t("here") }}</a></p>
+          </ion-card-content> 
+          <ion-item>
+            <ion-input clear-input='true' @keyup.enter="dateTimeFormat = $event.target.value; parse()" v-model="dateTimeFormat" :value="dateTimeFormat" />
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ sampleDateTime }}</ion-label>
+            <ion-badge color="warning">
+              <ion-label>{{ $t("Sample") }}</ion-label>
+            </ion-badge>
+          </ion-item>
+          <ion-button fill="clear" @click="updateDateTimeFormat()">
+            {{ $t("Save") }}
+            <ion-icon slot="end" :icon="saveOutline" />
+          </ion-button>
+        </ion-card>
+      </section>  
     </ion-content>
   </ion-page>
 </template>
 
 <script lang="ts">
-import { IonButton, IonContent, IonHeader,IonIcon, IonItem, IonLabel, IonMenuButton, IonPage, IonTitle, IonToolbar, modalController } from '@ionic/vue';
+import { IonAvatar, IonBadge, IonButton, IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle, IonContent, IonHeader,IonIcon, IonItem, IonInput, IonLabel, IonMenuButton, IonPage, IonTitle, IonToolbar, modalController } from '@ionic/vue';
 import { defineComponent } from 'vue';
-import { codeWorkingOutline, ellipsisVertical, personCircleOutline, timeOutline } from 'ionicons/icons'
+import { codeWorkingOutline, ellipsisVertical, personCircleOutline, openOutline, saveOutline, timeOutline } from 'ionicons/icons'
 import { mapGetters, useStore } from 'vuex';
 import { useRouter } from 'vue-router';
 import TimeZoneModal from '@/views/TimezoneModal.vue';
+import { DateTime } from 'luxon';
+import Image from '@/components/Image.vue';
 
 export default defineComponent({
   name: 'Settings',
   components: {
+    IonAvatar,
+    IonBadge,
     IonButton, 
+    IonCard,
+    IonCardContent,
+    IonCardHeader,
+    IonCardSubtitle,
+    IonCardTitle,
     IonContent, 
     IonHeader, 
     IonIcon,
     IonItem, 
+    IonInput,
     IonLabel, 
     IonMenuButton,
     IonPage,
     IonTitle, 
-    IonToolbar
+    IonToolbar,
+    Image
   },
   data() {
     return {
-      baseURL: process.env.VUE_APP_BASE_URL
+      baseURL: process.env.VUE_APP_BASE_URL,
+      sampleDateTime: '',
+      dateTimeFormat: ''
     };
   },
   computed: {
     ...mapGetters({
       userProfile: 'user/getUserProfile',
-      instanceUrl: 'user/getInstanceUrl'
+      instanceUrl: 'user/getInstanceUrl',
+      currentDateTimeFormat: 'user/getDateTimeFormat'
     })
   },
+  mounted(){
+    this.dateTimeFormat = this.currentDateTimeFormat
+    this.parse();
+  },
   methods: {
+    goToOms(){
+      window.location.href = `https://${this.instanceUrl}.hotwax.io/`;
```

Comment:
Open the oms in a new tab. Also check whether the instanceUrl contains `http` or not and handle the case accordingly.

Reply:
Made the requested change.